infrastructure/cdn-in-a-box/docker-compose.yml

```diff
@@ -58,10 +58,10 @@ services:
   # defined below
   trafficops:
     build:
-      context: ../..
-      dockerfile: infrastructure/cdn-in-a-box/traffic_ops/Dockerfile-go
+      context: .
+      dockerfile: traffic_ops/Dockerfile-go
       args:
-        TRAFFIC_OPS_RPM: infrastructure/cdn-in-a-box/traffic_ops/traffic_ops.rpm
+        TRAFFIC_OPS_RPM: traffic_ops/traffic_ops.rpm
```

Comment:
These path changes also need to be made to the `trafficops-go-nondebug` service in `/infrastructure/cdn-in-a-box/optional/docker-compose.debugging.yml`